vaadin-date-time-picker-flow-parent/vaadin-date-time-picker-flow/src/main/java/com/vaadin/flow/component/datetimepicker/DateTimePicker.java

```diff
@@ -347,6 +348,11 @@ public DateTimePicker(LocalDateTime initialDateTime, Locale locale) {
         setLocale(locale);
     }
 
+    @Override
+    protected final void setSynchronizedEvent(String event) {
```

Comment:
Should this method be final though? Why would we not allow it to be overridden?

Reply:
Even though there is no explicit mention of it, I thought this was meant to be more of an internal method so I decided to keep Sonar a little happier. But in any case I agree that it's not really relevant, so I reverted this change.